### ISSUE TITLE
Question about reproducing the result from cityscapes to acdc

### ISSUE THREAD

**Issue by ywher** [lhoyer/MIC#37]
Dear authors:

I am interested in your nice work and try to reproduce the results in your paper. I setup the environment according to issue 8 and can alomot reproduce the result from gta to cityscapes. But I can reproduce the result from cityscapes to acdc. Here is the training logs of from gta to cityscapes and from cityscapes to acdc. Could you please help me identify any issues during the training process?

Thanks a lot!
Weihao

[20230508_074602.log](https://github.com/lhoyer/MIC/files/11521868/20230508_074602.log)
[20230518_144316.log](https://github.com/lhoyer/MIC/files/11521869/20230518_144316.log)

**Comment by lhoyer** [comment]
Dear Weihao,

Thanks a lot for your interest in our work! You can find the training logs of CS to ACDC here: https://github.com/lhoyer/MIC/blob/master/seg/README.md#checkpoints. They come together with the checkpoints. Please note that the performance reported in the paper is on the ACDC test set while the mIoUs in the log are on the ACDC validation set.

Best,
Lukas
